python/python/tests/test_transaction.py

```diff
@@ -235,6 +239,14 @@ async def transaction_test(
     args.append(1)
     transaction.zremrangebylex(key8, InfBound.NEG_INF, InfBound.POS_INF)
     args.append(0)
+    transaction.zadd(key12, {"one": 1, "two": 2})
```

Comment:
lets just test on the existing sorted set key

Reply:
like this:
```suggestion
    transaction.zinterstore(key8, [key8, key8])
    args.append(0)
```

Reply:
I don't want it, as I want to know the outcome of the command.
In case of someone changing the content of it, it might fails.
In case of zinterstore, the result would be the length of the set in key8.
Let's keep it simple.